### DISCUSSION TITLE
I am unable to install makefile

### DISCUSSION THREAD

**Discussion by prakharmishra3956** [Cyfrin/foundry-full-course-cu#2822] [Q&A]
![Screenshot (81)](https://github.com/user-attachments/assets/3425b8e4-cd39-47a8-a6b8-5348039f8a12)

**Comment by EngrPips** [comment]
Hello @prakharmishra3956, It says, `your branch is ahead of origin/main by two commits` You can maybe push those commits to the origin/main branch and retry the `forge clean` command

**Comment by prakharmishra3956** [comment]
![Screenshot (82)](https://github.com/user-attachments/assets/69ab1bf3-2f40-4008-b6fc-b55eeb06306f)

Now It's showing this.

**Reply by EngrPips** [reply]
I am unsure what this error is. Are you sure there is no mistake in your `Makefile`
